### PR TITLE
Add new language or grammar instructions

### PR DESCRIPTION
**Pull request by DerekNonGeneric**
/cc @Trott

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,29 @@ npm install remark-preset-lint-node
...
+**Adding a new language to the highlight.js bundle**
+
+1. Visit [highlightjs.org/download](https://highlightjs.org/download/)
+2. Select [the pre-existing grammars](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs) and include any additional grammar(s)
+3. Download the custom `highlight.zip` package for Node.js
+3. Extract the downloaded `highlight.zip`
+4. Copy `highlight.pack.js` into the `doc/api_assets/` directory
+5. Update the date when the bundle was downloaded at [`doc/api_assets/README.md#highlightpackjs`](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs)
+6. Update [the pre-existing grammars](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs) at `doc/api_assets/README.md` to include the new grammar(s)
+7. PR the `nodejs/node` repo with the updated bundle
```
Comment (DerekNonGeneric) [inline README.md:43]:
```suggestion
1. PR this repo adding the language/grammar
1. Bump this package version, publish it
1. In [node-lint-md-cli-rollup](https://github.com/nodejs/node/tree/master/tools/node-lint-md-cli-rollup), bump the `remark-preset-lint-node` dependency 
1. In the `nodejs/node` repo, rebuild the Markdown linter (`make lint-md-rollup`)
1. PR the `nodejs/node` repo with the updated linter

**Adding a new language to the highlight.js bundle**

1. Visit [highlightjs.org/download](https://highlightjs.org/download/)
1. Select [the pre-existing grammars](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs) and include any additional grammar(s)
1. Download the custom `highlight.zip` package for Node.js
1. Extract the downloaded `highlight.zip`
1. Copy `highlight.pack.js` into the `doc/api_assets/` directory
1. Update the date when the bundle was downloaded at [`doc/api_assets/README.md#highlightpackjs`](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs)
1. Update [the pre-existing grammars](https://github.com/nodejs/node/tree/master/doc/api_assets#highlightpackjs) at `doc/api_assets/README.md` to include the new grammar(s)
1. PR the `nodejs/node` repo with the updated bundle
```

File: README.md [hunk 2]
```diff
@@ -15,3 +15,24 @@ npm install remark-preset-lint-node
 ```console
 npm test
 ```
+
+## Add new language or grammar
+
+**Adding the language to the linter**
```
Comment (nschonni) [inline README.md:21]:
```suggestion
### Adding the language to the linter
```

File: README.md [hunk 3]
```diff
@@ -15,3 +15,24 @@ npm install remark-preset-lint-node
...
+
+1. PR this repo adding the language/grammar
+1. Bump this package version, publish it
+1. In [node-lint-md-cli-rollup](https://github.com/nodejs/node/tree/master/tools/node-lint-md-cli-rollup), bump the `remark-preset-lint-node` dependency 
+1. In the `nodejs/node` repo, rebuild the Markdown linter (`make lint-md-rollup`)
+1. PR the `nodejs/node` repo with the updated linter
+
+**Adding a new language to the highlight.js bundle**
```
Comment (nschonni) [inline README.md:29]:
```suggestion
### Adding a new language to the highlight.js bundle
```

**Comment by Trott** [comment]
In theory, someone might think "language" refers to a spoken language, but the audience for this is very narrow and I think we'll know what it means, and I can't come up with a better word, and we can always change it later if someone thinks of a better word, and this run-on sentence is long enough so I'll stop.

**Pull request merged**